src/modules/tools/zprobe/ZGridStrategy.cpp

```diff
@@ -44,6 +44,10 @@
 
        leveling-strategy.ZGrid-leveling.wait_for_probe  true
 
+    The machine can be told to home in one of the following modes:
+
+       leveling-strategy.ZGrid-leveling.home_before_probe  2;    # 0: NOHOME  1: HOMEXY  2: HOMEXYZ (Default HOMEXYZ)
```

Comment:
I'm nitpicking here but why use 0, 1, 2 and not nohome, homexy, hmexyz in the config, it is much clearer, and I prefer that to cryptic numbers in the config.